pages/members/[id]/index.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import { useRouter } from 'next/router';
 import Profile from 'components/member-profile';
-
+import Layout from 'components/layout';
 const MemberProfile = () => {
   let { query } = useRouter();
-  return <Profile id={query.id} />;
+  return (
+    <Layout title={`${query.id} | Member Real Dev Squad`}>
```

Comment:
why id? It can the first name and `| RDS`.

Reply:
Then we need to request User data here just for first_name.
And requesting data from here and passing the data as prop won't be good.
because whille directly visiting `rds.com/members/shihabus` the component will not have any data.
Should we request the data twice?

Reply:
Can't you pass it as query along from the main page?